sdk/servicebus/Azure.Messaging.ServiceBus/tests/Primitives/ValueStopwatchTests.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Messaging.ServiceBus.Primitives;
+using NUnit.Framework;
+
+namespace Azure.Messaging.ServiceBus.Tests.Primitives
+{
+    public class ValueStopwatchTests
+    {
+        [Test]
+        public void IsActiveIsFalseForDefaultValueStopwatch()
+        {
+            Assert.False(default(ValueStopwatch).IsActive);
+        }
+
+        [Test]
+        public void IsActiveIsTrueWhenValueStopwatchStartedWithStartNew()
+        {
+            Assert.True(ValueStopwatch.StartNew().IsActive);
+        }
+
+        [Test]
+        public void GetElapsedTimeThrowsIfValueStopwatchIsDefaultValue()
+        {
+            var stopwatch = default(ValueStopwatch);
+            Assert.Throws<InvalidOperationException>(() => stopwatch.GetElapsedTime());
+        }
+
+        [Test]
+        public async Task GetElapsedTimeReturnsTimeElapsedSinceStart()
+        {
+            var stopwatch = ValueStopwatch.StartNew();
+            await Task.Delay(200);
+            Assert.True(stopwatch.GetElapsedTime().TotalMilliseconds > 0);
```

Comment:
Should this test verify that the elapsed time is no less than 200 msec?

Reply:
This is the original test from ASPNET Core

Reply:
While this is the original test, you could still validate that the time elapsed is no less than the expected value. Otherwise, anything positive would pass w/o necessarily validating that at least the expected time has elapsed.

Reply:
I have no objections of doing this once the question with the license is solved